src/sentry/models/release.py

```diff
@@ -60,7 +61,7 @@ class Release(Model):
     )
     # DEPRECATED
     project_id = BoundedPositiveIntegerField(null=True)
-    version = models.CharField(max_length=64)
+    version = models.CharField(max_length=VERSION_LENGTH)
     # ref might be the branch name being released
     ref = models.CharField(max_length=64, null=True, blank=True)
```

Comment:
Aren't you altering this field too?

Reply:
@mattrobenolt decided against it for now.